assets/scripts/app/StreetView.jsx

```diff
@@ -172,11 +168,14 @@ class StreetView extends React.Component {
     }
 
     this.setState({
-      buildingWidth: width
+      buildingWidth: width,
+      onResized: false
     })
   }
 
-  calculateBuildingPerspective = (el) => {
+  calculatePerspective = (el) => {
```

Comment:
I know you just renamed this already existing function, but I think we should rename it to be `updatePerspective`. `calculatePerspective` makes me think that it returns a value